back/api/lib/graphql/index.js

```diff
@@ -3,13 +3,19 @@
 const graphql = require('graphql').graphql;
 const Schema = require('./schema');
 
-module.exports = (query) => {
+module.exports = (eventQuery) => {
 
   // patch to allow queries from GraphiQL
   // like the initial introspectionQuery
-  if (query && query.hasOwnProperty('query')) {
-    query = query.query.replace("\n", ' ', "g");
+  let query, variables;
+
+  if (eventQuery && eventQuery.hasOwnProperty('query')) {
+    query = eventQuery.query.replace("\n", ' ', "g");
+  }
+
+  if (eventQuery && eventQuery.variables) {
+    variables = JSON.parse(eventQuery.variables);
   }
 
-  return graphql(Schema, query);
+  return graphql(Schema, query, null, variables);
```

Comment:
Shouldn't this be:
```javascript
graphql(Schema, query, null, null, variables);
```
?

The docs for `graphql`: http://graphql.org/docs/api-reference-graphql/#graphql

```
graphql(
  schema: GraphQLSchema,
  requestString: string,
  rootValue?: ?any,
  contextValue?: ?any,
  variableValues?: ?{[key: string]: any},
  operationName?: ?string
): Promise<GraphQLResult>
```

Anyway, this helped me in the right direction, I just needed two `null`s in between `query` and `variables`. Thanks!